source/demo/Application.js

```diff
@@ -35,7 +36,7 @@ class Application extends Component {
     const matches = window.location.search.match('component=(.+)')
     const activeComponent = matches && (COMPONENTS.includes(matches[1]) || HIGH_ORDER_COMPONENTS.includes(matches[1]))
       ? matches[1]
-      : 'VirtualScroll'
+      : 'WindowScroller'
```

Comment:
Whoops, meant to mention this before. We shouldn't change the demo default landing page, I don't think. No big deal though, I can always change this post-merge if needed. :)